libs/platform/src/lib/components/form/combobox/combobox/combobox.component.scss

```diff
@@ -1,3 +1,6 @@
+@import "~@angular/cdk/overlay-prebuilt";
```

Comment:
This shoudl not be part of the component, overlay is used on other places, it should be some global imports